src/pages/task/metadata/index.tsx

```diff
@@ -277,6 +278,12 @@ export const Metadata: React.VFC<Props> = ({
           ))}
         </span>
       ) : null}
+      {resetWhenFinished && (
+        <P2 data-cy="reset-when-finished-container">
+          This task is scheduled to restart when all of the tasks in its group
+          have finished.
```

Comment:
lol I know we say "task" a lot in this sentence but I feel like saying `task group` might be a little clearer (other stuff is just suggestion, I don't feel strongly about it)

```suggestion
          This task will restart when all of the tasks in its task group
          have finished running.
```